ReactWindows/ReactNative.Shared/CoreModulesPackage.cs

```diff
@@ -43,12 +47,11 @@ public IReadOnlyList<INativeModule> CreateNativeModules(ReactContext reactContex
             using (Tracer.Trace(Tracer.TRACE_TAG_REACT_BRIDGE, "createUIManagerModule").Start())
             {
                 var viewManagerList = _reactInstanceManager.CreateAllViewManagers(reactContext);
-                var layoutActionQueue = new LayoutActionQueue(reactContext.HandleException);
                 uiManagerModule = new UIManagerModule(
-                    reactContext, 
+                    reactContext,
                     viewManagerList,
                     _uiImplementationProvider,
-                    layoutActionQueue);
+                    _layoutActionQueue);
```

Comment:
>_layoutActionQueue [](start = 20, length = 18)

Hopefully this won't be necessary, if we can figure out why the MainView is not accessible after calling an async function in the background.